pkg/queue/manager_test.go

```diff
@@ -798,7 +798,7 @@ func TestHeadsAsync(t *testing.T) {
 				// Remove the initial workload from the manager.
 				mgr.Heads(ctx)
 				go func() {
-					mgr.RequeueWorkload(ctx, workload.NewInfo(&wl))
+					mgr.RequeueWorkload(ctx, workload.NewInfo(&wl), true)
```

Comment:
Even though we are using StrictFIFO here, false is more appropriate (we want immediate requeue).